spec/fixtures/state_file/persona_approved_outputs/2023/az/johnny_return_xmls/xml/submission.xml

```diff
@@ -263,18 +263,17 @@
         <EmployersUseCd>Q</EmployersUseCd>
         <EmployersUseAmt>0</EmployersUseAmt>
       </EmployersUseGrp>
-      <W2StateLocalTaxGrp>
-        <W2StateTaxGrp>
-          <StateAbbreviationCd>AZ</StateAbbreviationCd>
-          <EmployerStateIdNum>12345</EmployerStateIdNum>
-          <StateWagesAmt>45000</StateWagesAmt>
-          <StateIncomeTaxAmt>400</StateIncomeTaxAmt>
-          <W2LocalTaxGrp>
-            <LocalWagesAndTipsAmt>0</LocalWagesAndTipsAmt>
-            <LocalIncomeTaxAmt>0</LocalIncomeTaxAmt>
-          </W2LocalTaxGrp>
-        </W2StateTaxGrp>
-      </W2StateLocalTaxGrp>
+      <W2StateLocalTaxGrp><W2StateTaxGrp>
+  <StateAbbreviationCd>AZ</StateAbbreviationCd>
+  <EmployerStateIdNum>12345</EmployerStateIdNum>
+  <StateWagesAmt>45000</StateWagesAmt>
+  <StateIncomeTaxAmt>400</StateIncomeTaxAmt>
+  <W2LocalTaxGrp>
+    <LocalWagesAndTipsAmt>0</LocalWagesAndTipsAmt>
+    <LocalIncomeTaxAmt>0</LocalIncomeTaxAmt>
+  </W2LocalTaxGrp>
+</W2StateTaxGrp>
+</W2StateLocalTaxGrp>
```

Comment:
i think i tried a little too hard and wasted a little too much time trying to make this output the w2s with the right formatting because the contents haven't changed. potentially not worth it? i'm wondering:
A) have we just never built the persona outputs with combined w2s (did they all come from the df xml? i think maybe they did) which is why this hasn't come up
B) should i manually reformat it or is that dumb because the next person who regenerates these will have to do it too